test/core/utils.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Test: core/utils.js
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const web3 = require('../test_lib/web3.js');
+
+const VOTE_MESSAGE_TYPEHASH = web3.utils.sha3(
+     "VoteMessage(bytes20 coreIdentifier,bytes32 transitionHash,bytes32 source,bytes32 target,uint256 sourceHeight,uint256 targetHeight)"
+);
+
+function Utils() {
+
+}
+
+Utils.prototype = {
+    /**
+     * @param {string} address The address of the account that signs the vote.
+     * @param {Object} vote The vote object to sign.
+     * @returns {Object} The signature of the vote (r, s, and v).
+     */
+    signVote : async function (address, vote) {
```

Comment:
Do we maybe already have similar logic somewhere else that should be merged here? Or maybe move this into `test/test_lib/meta_block.js`?